packages/vue-components/src/Navbar.vue

```diff
@@ -363,12 +363,18 @@ export default {
         display: inline-block;
     }
 
+    .navbar-brand > img,
+    svg {
```

Comment:
```suggestion
    .navbar-brand > svg {
```

Should this have the `.navbar-brand` selector as well? If not this will select all `svg` elements.